src/Cassandra/ExecutionProfile.cs

```diff
@@ -0,0 +1,77 @@
+﻿// 
+//       Copyright (C) 2019 DataStax Inc.
+// 
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+// 
+//       http://www.apache.org/licenses/LICENSE-2.0
+// 
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+// 
+
+using System;
+using Cassandra.ExecutionProfiles;
+
+namespace Cassandra
+{
+    public class ExecutionProfile
```

Comment:
I agree with extracting an interface. Where should the `ExecutionProfile.Builder` method go in this case? We either remove it and have users just do `new ExecutionProfileBuilder()` or we can move it to the cluster builder class? `Builder.ExecutionProfileBuilder()`?

Reply:
Maybe `Cluster.ExecutionProfileBuilder` is also a possibility since the method to create the cluster builder is also on the `Cluster` class. For now I've added it to the `Builder` class: `Builder.ExecutionProfileBuilder()`, let me know your thoughts